src/docs/reference/modules/Media/README.md

```diff
@@ -501,12 +505,32 @@ To set up indexing for Media do the following:
 3. Configure the new field to be used for search. You can do this from the admin under Search, Settings, Search, and adding the name of the new field under "Default search fields" (arriving at something like "Content.ContentItem.FullText, BlogPost.File.MediaText, BlogPost.File.FileText").
 4. Try searching for content only available in the Media Text of selected media files, or referenced PDF files. You should see corresponding results.
 
+## Secure Media
+
+The Secure Media feature enhances security and control over media files within the Media module. 
+
+When enabled, administrators can set view permissions for the root media folder and each first-level folder within the media root. This allows for restricting access to media folders based on user roles, ensuring that only authorized users can view or manage media files within specific folders.
+
+New permissions to allow users to view their own media files, view media files uploaded by others, or both are created too. You can manage these among the other permissions with the [Roles module](../Roles/README.md).
```

Comment:
```suggestion
New permissions to allow users to view their own media files, view media files uploaded by others or both are created too. You can manage these among the other permissions with the [Roles module](../Roles/README.md).
```

Reply:
I think the comma is correct...

Reply:
A grammar tool helps me on that :)

Reply:
Both is grammatically correct as far as I know, mine emphasizes the fact that there are three scenarios:

- Viewing one's own media files,
- Viewing media files uploaded by others,
- Viewing both.

I would rather keep it for clarity.